lambdas/indexer/app.py

```diff
@@ -65,8 +75,17 @@ def health(keys: Optional[str] = None):
             body = health.as_json(keys)
         except RequirementError:
             raise BadRequestError('Invalid health keys')
-        return Response(body=json.dumps(body),
-                        status_code=200 if body['up'] else 503)
+    return Response(body=json.dumps(body),
+                    status_code=200 if body['up'] else 503)
+
+
+@app.schedule('rate(1 minute)', name=config.indexer_cache_health_lambda_basename)
+def generate_health_object(event: chalice.app.CloudWatchEvent):
+    assert event is not None  # avoid linter warning
+    indexer_health = Health('indexer')
```

Comment:
```suggestion
    health = Health('indexer')
```

in order to reduce the differences between the two copies of this method. That will make extracting a single method easier in the future.